extensions/amp-story/amp-story-bookend.md

```diff
@@ -0,0 +1,304 @@
+---
+$category@: presentation
+formats:
+  - websites
+  - stories
+teaser:
+  text: The end screen of an AMP story, showcasing sharing and related content.
+---
+<!--
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## Bookend: `amp-story-bookend`
```

Comment:
All of the new component docs should be an H1 and have an introductory table outlining `required script`, `supported layouts`, and `examples` (if applicable). See [amp-accordion example](https://raw.githubusercontent.com/ampproject/amphtml/master/extensions/amp-accordion/amp-accordion.md).